website/docs/developer/deployment-guide/testing_deployment.mdx

```diff
@@ -0,0 +1,28 @@
+---
+id: testing_deployment
+title: Testing Deployment
+description: Provides information on testing ORCA code deployment.
+---
+
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The grey fox jumped over the red fern chasing the pink pig.
+
+<MyImage
+    imageSource={useBaseUrl('static/example_test_data/DR.jpg')}
+    imageAlt="A fox chasing a pig."
+    zoomInPic={useBaseUrl('static/img/zoom-in.svg')}
+    zoomOutPic={useBaseUrl('static/img/zoom-out.svg')}
+    resetPic={useBaseUrl('static/img/zoom-pan-reset.svg')}
+/>
+
+This document is currently under construction.
+
+## Example Data
+Below are several basic files in formats used by ORCA.
+They can be used to check deployment functionality.
+- <a href={useBaseUrl('static/example_test_data/DR.jpg')} target="_blank" download>JPG</a>
```

Comment:
Remove `static/` from your URL path in all the links. When Docusaurus compiles and directories in the static directory get moved to the root. On a build, a path not found error will occur if you specify `static/exaple_test_data/...` instead of `example_teast_data/...`.